### ISSUE TITLE
New steering wheel heater funcionallity

### ISSUE THREAD

**Issue by Urkman** [timdorr/tesla-api#684]
Is there already something known about the new steering wheel heater API?

**Comment by timdorr** [comment]
https://tesla-api.timdorr.com/vehicle/commands/climate#post-api-1-vehicles-id-command-remote_steering_wheel_heater_request

**Comment by itsMeDavidV** [comment]
@timdorr I think @Urkman is referring to the ability to set steering heater at multiple levels which the param for that is still not doc bc its just been added.

**Comment by Urkman** [comment]
@itsMeDavidV  Yes, that what I asked for... Now there are Levels Off, Auto, Low and High...

**Comment by timdorr** [comment]
I'm not sure what the new parameters are yet. PRs are appreciated, though!

**Comment by core-hacked** [comment]
Most likely simillar to: https://tesla-api.timdorr.com/vehicle/commands/climate#post-api-1-vehicles-id-command-set_climate_keeper_mode

eg. 
| Number | Mode |
|--------|------|
| 0      | off  |
| 1      | auto |
| 2      | low  |
| 3      | high |

and the needed parameter could be something along the lines of `steering_wheel_heater_mode` or `steering_wheel_mode` etc.

Also on a side note, is this feature actually present in app? From what I have seen it is in-car but I might be wrong.

**Comment by Urkman** [comment]
@core-hacked Yes, it's currently only available in-car...
So I think, we need to wait :)

**Comment by core-hacked** [comment]
@timdorr currently in-car only, please close. Kind Regards!

**Comment by Urkman** [comment]
Why close? I think, this will be available shortly...

**Comment by core-hacked** [comment]
> Why close? I think, this will be available shortly...

When it will be available the issue can be re-opened. Tesla might just decide not to include it. Why should the issue just sit around, when we have no information on when, how and what?

**Comment by timdorr** [comment]
Yeah, closing until it's available.

**Comment by core-hacked** [comment]
#696 | The endpoints for this have been put into the `ownerapi_endpoints.json`.

**Comment by itsMeDavidV** [comment]
@core-hacked sweet! thanks for pointing this out

**Comment by Megabytemb** [comment]
Hey, Gentle bump. Does anyone have the body required for `REMOTE_AUTO_STEERING_WHEEL_HEAT_CLIMATE_REQUEST `?

**Comment by Megabytemb** [comment]
Hey, still hoping someone can provide the body for this?

**Comment by Megabytemb** [comment]
For anyone coming here, The requests are as follows

```
Remote_auto_steering_wheel_heat_climate_request
{
  "on": true
}

Remote_steering_wheel_heat_level_request

{
  "level": 0
}
{
  "level": 3
}
{
  "level": 1
}

```
